index.html

```diff
@@ -882,16 +882,30 @@ <h3>Issuing</h3>
       <section>
         <h4>Issue Credential</h4>
         <p>
+This endpoint is used to issue a [=verifiable credential=].
         </p>
 
         <p class="note" title="Issued credential media types">
-          An `EnvelopedVerifiableCredential` can be returned in the `IssueCredentialSuccess` response
-          in order to issue credentials with a media type other than `application/vc`,
-          such as `application/vc+sd-jwt`.
+An `EnvelopedVerifiableCredential` can be returned in the response in order to
+issue credentials with a media type other than `application/vc`, such as
+`application/mdoc` or `application/vc+sd-jwt`.
```

Comment:
I wonder if we should also explicitly mention the VCB media type here for VCs issued inside a data container that will travel in a barcode (e.g., QR codes, PDF417, so `application/vcb;barcode-format=qr_code` / `application/vcb;barcode-format=pdf417`). This should also be mentioned on the verification side (and I imagine the same mention of mdoc, etc. might be required there if not already present).

As a separate consideration, we could mention using an image media type for expressing an image of such a code that contains a VCB.

Reply:
Added in 1f63c7376fba8d12186a64ec4110d3e9d39d2a10.

The language is a bit rough, and this raises the question around what should be returned in `credential`.